/* -*- Mode:protobuf; indent-tabs-mode:nil; -*- */
// More information: https://developers.google.com/protocol-buffers/docs/cpptutorial

syntax = "proto3";

package ndn.ndvr.proto;

message DvInfo {
  message NextHop {
    repeated string router_id = 1;
    uint32 cost = 2;
  }
  message Entry {
    string prefix = 1;
    uint64 seq = 2;
    string originator = 3;
    NextHop next_hops = 4;
  }
  repeated Entry entry = 1;
}
